FRC1076-2016-Competition/src/org/usfirst/frc/team1076/udp/SensorData.java

```diff
@@ -30,6 +33,9 @@ public SensorData(IChannel channel, FieldPosition position, IGyro gyro) {
 
 	@Override
 	public void interpretData() {
+		// Vision gets staler and staler over time, but will reset every time
+		// we actually see a message from the vision system.
+		visionStaleness++;
 		while (receiver.hasMessage()) {
```

Comment:
We don't ask for them, they come in as fast as the vision system can produce them. They should only get stale if we can't actually see the tower.